libraries/rush-lib/src/logic/operations/CacheableOperationPlugin.ts

```diff
@@ -181,6 +181,8 @@ export class CacheableOperationPlugin implements IPhasedCommandPlugin {
             }
           }
 
+          logCobuildBuildPlan(disjointSet, terminal, this._buildCacheContextByOperation);
```

Comment:
should this logging be feature flagged?

Reply:
Hmm, I think you're right. My goal was to have this be visible for teams that are using cobuilds to see where clustering might be going wrong, but I think a better solution there would be to emit this as telemetry that could be tracked across builds.

Reply:
updated these to debug logs, is it possible to emit telemetry from plugins?